lte/gateway/python/magma/enodebd/data_models/transform_for_magma.py

```diff
@@ -65,18 +65,20 @@ def bandwidth(bandwidth_rbs: Union[str, int, float]) -> float:
     BaiCells eNodeB uses 'n6'. Need to resolve this.
 
     Args:
-        bandwidth_rbs (str): Bandwidth in number of RBs
+        bandwidth_rbs (str, float): Bandwidth in number of RBs
     Returns:
         str: Bandwidth in MHz
     """
-    if bandwidth_rbs in BANDWIDTH_RBS_TO_MHZ_MAP:
+    if isinstance(bandwidth_rbs, str) and \
+            bandwidth_rbs in BANDWIDTH_RBS_TO_MHZ_MAP:
```

Comment:
In this case, I would prefer to have a `@no_type_check` for this method rather than all these `isinstance` checks that don't really make sense. It's a case where mypy seems to make the code worse.